server.js

```diff
@@ -9,18 +9,28 @@ var html = `
 <svg>
   <style>
     * {
-      font-weight: 300;
-      font-size: 12px;
-      font-family: 'Whitney';
```

Comment:
if we don't want to use `Whitney` anymore, we can remove the `.fonts` directory which was adding the font to Heroku